trl/trainer/utils.py

```diff
@@ -185,10 +186,10 @@ def torch_call(self, examples: List[Union[List[int], Any, Dict[str, Any]]]) -> D
 
                 if len(response_token_ids_idxs) == 0:
                     warnings.warn(
-                        f"Could not find response key `{self.response_template}` in the "
-                        f'following instance: {self.tokenizer.decode(batch["input_ids"][i])} '
-                        f"This instance will be ignored in loss calculation. "
-                        f"Note, if this happens often, consider increasing the `max_seq_length`."
+                        f"Could not find response key `{self.response_template}` in the following instance: "
+                        f"{self.tokenizer.decode(batch["input_ids"][i])}. This instance will be ignored in loss "
+                        "calculation. Note, if this happens often, consider increasing the `max_seq_length`.",
+                        UserWarning,
```

Comment:
Warnings should not indicate normal behavior.
Use the appropriate warning type.

This warning can still occur in normal behavior. Any idea?